easybuild/easyblocks/g/gromacs.py

```diff
@@ -203,6 +205,65 @@ def configure_step(self):
                 # to avoid that GROMACS finds and uses a system-wide CUDA compiler
                 self.cfg.update('configopts', "-DGMX_GPU=OFF")
 
+        # CP2K detection
+        # enable CP2K support if CP2K is listed as a dependency
+        # and CP2K support is either explicitly enabled (cp2k = True) or unspecified ('cp2k' not defined)
+        cp2k_root = get_software_root('CP2K')
+        if self.cfg['cp2k'] and not cp2k_root:
+            msg = "CP2K support has been requested but CP2K is not listed as a dependency."
+            raise EasyBuildError(msg)
+        elif cp2k_root and self.cfg['cp2k'] is False:
+            self.log.info('CP2K was found, but compilation without CP2K has been requested.')
+            cp2k_root = None
+
+        if cp2k_root:
+            if LooseVersion(self.version) < LooseVersion('2022'):
+                msg = 'CP2K support is only available for GROMACS 2022 and newer.'
+                raise EasyBuildError(msg)
+            elif LooseVersion(get_software_version('CP2K')) < LooseVersion('8.1'):
```

Comment:
This can be a simple "if", then one can put the 
```
cp2k_version = get_software_version('CP2K')
```
before this line.

Reply:
done in 6b1f77d88